pallets/identity/src/lib.rs

```diff
@@ -601,13 +593,15 @@ decl_module! {
 
             // Also set current_did roles when acting as a signing key for target_did
             // Re-dispatch call - e.g. to asset::doSomething...
-            let new_origin = frame_system::RawOrigin::Signed(sender).into();
+            // let new_origin = frame_system::RawOrigin::Signed(sender).into();
 
-            proposal.dispatch(new_origin)
+            // TODO: Handle dispatch properly
+            //proposal.dispatch(new_origin)
```

Comment:
I am not sure if we are using this fn, but commented lines disable its functionality.

Reply:
Yeah, we are not using this functionality, I commented it because it needs weight handling and return type change that I will do in another task, similar changes in other places as well. It is a kind of functionality added as well.